model/workload/workload.go

```diff
@@ -142,36 +244,42 @@ func Enforce(w *tw.RDTWorkLoad) *rmderror.AppError {
 		}
 	}
 
-	var resAss *resctrl.ResAssociation
-	var grpName string
+	if mbaInfo.MbaOn {
+		mbaStr := "100"
+		if er.Type == rdtpool.Guarantee && w.MbaPercentage != nil {
+			mbaStr = strconv.FormatUint((uint64)(*w.MbaPercentage), 10)
+		}
+		for k, v := range candidate {
+			if ok := mbaTarget[k]; ok {
+				newCandidate[k] = mixedCandidate{v, &mbaStr}
+			} else {
+				newCandidate[k] = mixedCandidate{v, &maxMba}
```

Comment:
This line 256 is optional

Reply:
Skip to assign maxMba is okay, but the variable "v" is necessary, it is CAT settings.
I suggest that it is more readable to keep this maxMba assignment

Reply:
Alright lets keep it. Please merge close on the other comments as possible.